tests/test_download_firmware.py

```
@@ -3187,3 +3272,312 @@ def test_cleanup_superseded_prereleases_hash_suffixed_tag(
+            patch.object(
+                downloader.cache_manager,
+                "get_repo_directories",
+                side_effect=requests.RequestException("Second scan failed"),
+            ),
+            patch.object(
+                downloader,
+                "_download_prerelease_assets",
+                return_value=([], [], False),
+            ),
+        ):
+            results, failed, latest, summary = (
```

Ruff (0.15.12)

[warning] Unpacked variable `summary` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)